keyboards/torn/keymaps/kinesish/config.h

```diff
@@ -0,0 +1,3 @@
+#define TAPPING_TERM 200
```

Comment:
This needs a license header, and an include guard (`#pragma once`)

Reply:
done and done!